### DISCUSSION TITLE
Entity Linker model's architecture

### DISCUSSION THREAD

**Discussion by amirj** [explosion/spaCy#8162] [Help: Coding & Implementations] [answered]
It seems that Entity Linker leverages different architectures, e.g. [here](https://github.com/explosion/spaCy/blob/202943bc8cc1491f05e8c8563d9b280b342dc827/spacy/pipeline/entity_linker.py#L26). I can't find these architectures in [the source code](https://github.com/explosion/spaCy/blob/master/spacy/pipeline/entity_linker.py); may I ask you to let me know where these architectures are defined?

**Comment by polm** [comment, marked as answer]
Architectures are defined in `spacy.ml.models`:

https://github.com/explosion/spaCy/blob/ff3fed06cf5733647122718a0bf7d842a1d497a0/spacy/ml/models/entity_linker.py#L11

If you see a registry reference you don't recognize, you can always use `git grep` or the github repo search for the name to find it.
